.github/workflows/create-package.yml

```diff
@@ -0,0 +1,224 @@
+
+                if [[ "${INCLUDE_DEPENDENCIES}" == "true" ]]; then
+                  create-package \
+                    --source ${SOURCE_PATH:-.} \
+                    --cache-location "${HOME}"/carton-cache \
+                    --destination "${HOME}"/buildpack \
+                    --include-dependencies \
+                    --version "${VERSION}"
+                else
+                  create-package \
+                    --source ${SOURCE_PATH:-.} \
+                    --destination "${HOME}"/buildpack \
+                    --version "${VERSION}"
+                fi
+
+                PACKAGE_FILE=${SOURCE_PATH:-.}/package.toml
+                [[ -e ${PACKAGE_FILE} ]] && cp ${PACKAGE_FILE} "${HOME}"/package.toml
+                printf '[buildpack]\nuri = "%s"\n\n[platform]\nos = "%s"\n' "${HOME}"/buildpack "${OS}" >> "${HOME}"/package.toml
+              env:
+                INCLUDE_DEPENDENCIES: "false"
+                OS: linux
+                SOURCE_PATH: ""
+                VERSION: ${{ steps.version.outputs.version }}
+            - name: Package Buildpack
+              id: package
+              run: |-
+                #!/usr/bin/env bash
+
+                set -euo pipefail
+
+
+                PACKAGE_LIST=($PACKAGES)
+                # Extract first repo (Docker Hub) as the main to package & register
+                PACKAGE=${PACKAGE_LIST[0]}
+
+                if [[ "${PUBLISH:-x}" == "true" ]]; then
+                  pack buildpack package \
+                    "${PACKAGE}:${VERSION}" \
+                    --config "${HOME}"/package.toml \
+                    --publish
+
+                  if [[ -n ${VERSION_MINOR:-} && -n ${VERSION_MAJOR:-} ]]; then
+                    crane tag "${PACKAGE}:${VERSION}" "${VERSION_MINOR}"
+                    crane tag "${PACKAGE}:${VERSION}" "${VERSION_MAJOR}"
+                  fi
+                  crane tag "${PACKAGE}:${VERSION}" latest
+                  echo "::set-output name=digest::$(crane digest "${PACKAGE}:${VERSION}")"
+
+                  # copy to other repositories specified
+                  for P in "${PACKAGE_LIST[@]}"
+                    do
+                      if [ "$P" != "$PACKAGE" ]; then
+                        crane copy "${PACKAGE}:${VERSION}" "${P}:${VERSION}"
+                        if [[ -n ${VERSION_MINOR:-} && -n ${VERSION_MAJOR:-} ]]; then
+                           crane tag "${P}:${VERSION}" "${VERSION_MINOR}"
+                           crane tag "${P}:${VERSION}" "${VERSION_MAJOR}"
+                        fi
+                        crane tag "${P}:${VERSION}" latest
+                      fi
+                    done
+
+                else
+                  pack buildpack package \
+                    "${PACKAGE}:${VERSION}" \
+                    --config "${HOME}"/package.toml \
+                    --format "${FORMAT}"
+                fi
+              env:
+                PACKAGES: docker.io/buildpacks/profile
```

Comment:
```suggestion
                PACKAGES: docker.io/buildpacksio/profile
```

Reply:
@dmikusa-pivotal I have added the appropriate secrets to the repository. Can you please make sure that the dockerhub org is buildpacksio and not buildpacks? Thanks.

Reply:
OK, creds look good. I fixed a few and have double-checked that all of the images are `docker.io/buildpacksio` and code references `github.com/buildpacks`.